### PR TITLE
Update old screenshots and videos

### PR DESCRIPTION
**Pull request by lucasponce**
Fixes https://github.com/kiali/kiali/issues/3054

It updates Features section with new screenshots and videos.

I haven't changed too much the existing wording on purpose, this PR is too big and I'd prefer to merge and if someone better familiar with a feature wants to update the wording paragraph/description, please do it in a following PR not on this.

Also, content wise I tried to allign a little bit the images adjusting to width in most cases for better visualization and consistency.

Note, that this only updates the Features section, but other areas may need following PRs.

Important, this is for 1.22 but I write the content on edge, please @cfcosta @jshaughn tell me what change we should do to make it work on 1.22.x (I suspect that put things on that folder, but I didn't want to mess more this one).

### CODE REVIEW

File: content/documentation/edge/features/index.adoc [hunk 1]
```diff
@@ -17,20 +18,27 @@ type: "features"
...
 
-== Overview
-
 Kiali helps you define, validate, and observe the connections and microservices of your Istio service mesh.
 
-Kiali works with Istio in OpenShift or Kubernetes. It visualizes the service mesh topology and provides visibility into features like request routing, circuit breakers, request rates, latency and more. Kiali offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads.
+Kiali works with Istio in Kubernetes distributions. It visualizes the service mesh topology and provides visibility into features like request routing, circuit breakers, request rates, latency and more. Kiali offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads.
 
-icon:lightbulb[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} Kiali also includes Jaeger Tracing to provide distributed tracing by default.
+Kiali also helps to operate the Service Mesh providing Wizards to apply common traffic patterns an generate automatically Istio configuration.
```
Comment (jshaughn) [inline content/documentation/edge/features/index.adoc:25]: operate --> manage (optional)
an --> and (required)
generate automatically --> automatically generate (optional)

File: content/documentation/edge/features/index.adoc [hunk 2]
```diff
@@ -143,55 +188,117 @@ Kiali provides filtered list views of all your service mesh definitions. Each vi
...
 ++++
 
 {empty} +
 
-==== Detail: Metrics
-Each detail view provides predefined metric dashboards. The metric dashboards provided are tailored to the relevant application, workload or service level.
+==== Detail: Overview
 
-Application and workload detail views show request and response metrics such as volume, duration, size, or tcp traffic. The traffic can also be viewed for either inbound or outbound traffic.
+The overview tab provides detail information, graph details and health overview for a specific Application, Workload or Service.
```
Comment (jshaughn) [inline content/documentation/edge/features/index.adoc:219]: graph details --> sparkline graph (optional)

File: content/documentation/edge/features/index.adoc [hunk 3]
```diff
@@ -143,55 +188,117 @@ Kiali provides filtered list views of all your service mesh definitions. Each vi
...
 
 {empty} +
 
-==== Detail: Services
-The service detail view shows the user the workloads running the service. It also shows the Istio traffic routing configuration, VirtualServices and DestinationRules, associated with the service.
+==== Detail: Traffic
+
+The traffic tab collects all connections from and to a specific Application, Workload and/or Service.
+
+It groups them in Inbound and Outbound tables prividing traffic details and links to specific detailed metrics.
```
Comment (jshaughn) [inline content/documentation/edge/features/index.adoc:261]: prividing --> providing (required)

File: content/documentation/edge/features/index.adoc [hunk 4]
```diff
@@ -143,55 +188,117 @@ Kiali provides filtered list views of all your service mesh definitions. Each vi
...
 ++++
 
 {empty} +
 
-==== Detail: Workloads
-Kiali performs several validations on workload configuration:
+==== Detail: Logs
 
-* Are Istio sidecars deployed?
+The Workload detail offer a special Logs tab. Kiali allows to examine at same time application containers and sidecar logs of a selected Pod.
```
Comment (jshaughn) [inline content/documentation/edge/features/index.adoc:273]: (optional/suggested)  for second sentence:

Kiali allows side-by-side viewing of both the application and the proxy logs of a selected pod.

File: content/documentation/edge/features/index.adoc [hunk 5]
```diff
@@ -143,55 +188,117 @@ Kiali provides filtered list views of all your service mesh definitions. Each vi
...
 
-* Are proper *app* and *version* labels assigned?
+The logs tab adds additional filtering cappabilities to show or hide specific lines.
```
Comment (jshaughn) [inline content/documentation/edge/features/index.adoc:275]: cappabilities --> capabilities (required)

File: content/documentation/edge/features/index.adoc [hunk 6]
```diff
@@ -279,24 +445,90 @@ This wizard helps user to partially or totally stop traffic for a service. You c
...
+   <img src="/images/documentation/features/wizard-suspend-traffic-v1.22.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
 ++++
 When there is traffic for some workload, the wizard maps a weighted rule; when there is not traffic, an abort rule is coded in the pair of Istio resources VirtualService and DestinationRule generated.
 
 ==== Advanced Options
-All previous wizards have an Advanced Options section where user can define specific configuration for TLS and LoadBalancing.
+All previous wizards have an Advanced Options section where user can define specific configuration for TLS, LoadBalancing, Security and Gateways.
```
Comment (jshaughn) [inline content/documentation/edge/features/index.adoc:459]: user --> the user (required)

File: content/documentation/edge/features/index.adoc [hunk 7]
```diff
@@ -279,24 +445,90 @@ This wizard helps user to partially or totally stop traffic for a service. You c
...
+  <a class="image-popup-fit-height" href="/images/documentation/features/create-istio-config-v1.22.0.png" title="Service Details Actions">
+   <img src="/images/documentation/features/create-istio-config-v1.22.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+==== Istio Security Forms
+
+Kiali allows to create AuthorizationPolicy, PeerAuthentication and RequestAuthentication Istio resources from the Create New Istio Config forms:
```
Comment (jshaughn) [inline content/documentation/edge/features/index.adoc:493]: to create --> creation of (optional)

File: content/documentation/edge/features/index.adoc [hunk 8]
```diff
@@ -279,24 +445,90 @@ This wizard helps user to partially or totally stop traffic for a service. You c
...
+      <a class="image-popup-fit-height" href="/images/documentation/features/form-request-authentication-v1.22.0.png" title="Custom JVM Metrics">
+          <img src="/images/documentation/features/form-request-authentication-v1.22.0.png" style="width: 660px; display:inline;margin: 0 auto;" />
+      </a>
+    </span>
+</div>
+++++
+
+==== Istio Traffic Forms
+
+Kiali uses Istio Wizards to generate Istio Traffic config for a specific Service, but Kiali also allows to create Gateway and Sidecar Istio resources for more generic scenarios:
```
Comment (jshaughn) [inline content/documentation/edge/features/index.adoc:517]: to create --> creation of (optional)

**Comment by lucasponce** [comment]
@jshaughn Thanks. I've addressed your comments.

**Pull request merged**